components/com_finder/tmpl/search/default_form.php

```diff
@@ -27,6 +27,9 @@
 <form action="<?php echo Route::_($this->query->toUri()); ?>" method="get" class="js-finder-searchform">
 	<?php echo $this->getFields(); ?>
 	<fieldset class="com-finder__search word mb-3">
+    <legend class="com-finder__search-legend sr-only">
+	    <?php echo Text::_('COM_FINDER_SEARCH_FORM_LEGEND'); ?>
+    </legend>
```

Comment:
```suggestion
	<legend class="com-finder__search-legend sr-only">
		<?php echo Text::_('COM_FINDER_SEARCH_FORM_LEGEND'); ?>
	</legend>
```
Please use tabs for indentation.